go/src/koding/klient/machine/machinegroup/mount.go

```diff
@@ -0,0 +1,92 @@
+package machinegroup
+
+import (
+	"errors"
+	"fmt"
+
+	"koding/klient/machine"
+	"koding/klient/machine/mount"
+)
+
+// HeadMountRequest defines machine group head mount request.
+type HeadMountRequest struct {
+	// ID is a unique identifier for the remote machine.
+	ID machine.ID `json:"id"`
+
+	// Mount describes the mount to be headed.
+	Mount mount.Mount `json:"mount"`
+}
+
+// HeadMountResponse defines machine group head mount response.
+type HeadMountResponse struct {
+	// ExistMountID is not empty when mount to a given remote folder already exists.
+	ExistMountID mount.ID `json:"existMountID"`
```

Comment:
According to the doc it can be empty - how about making it `json:"existMountID,omitempty"`?